### PR TITLE
Add checkbox to eliminate page breaks after groups (Issue #47)

### PR DESCRIPTION
**Pull request by anchovie91471**
- Controller.php: - Changed $options to a multidimensional array to allow default value to be set for checkboxes. - Added a controller function (for development purposes) to allow the PDF view to be rendered as a web view (requires an env variable of PDFPREVIEW=true to be set)
- home.blade.php: - Added a check PDFPREVIEW=true to set the preview url in case if dev/debugging - Changed $options loop to accommodate the array changes in Controller.php
- pdf.blade.php: Check for the pagebreaks option and render css if checked=true
- web.php (route): Added route for PDF preview as webpage

### CODE REVIEW

File: app/Http/Controllers/Controller.php [hunk 1]
```diff
@@ -99,7 +99,14 @@ public function home()
             'stream' => 'Stream',
         ];
         $options = [
-            'legend' => 'Meeting Types Legend',
+            'legend' => [
+                'label' => 'Meeting Types Legend',
+                'checked' => false,
+                ],
```
Comment (joshreisner) [inline app/Http/Controllers/Controller.php:105]: would be great to get format with PSR-12

File: app/Http/Controllers/Controller.php [hunk 2]
```diff
@@ -432,4 +439,323 @@ public function pdf()
 
         return ($stream) ? $pdf->stream() : $pdf->download('directory.pdf');
     }
+
+    public function pdfpreview()
```
Comment (joshreisner) [inline app/Http/Controllers/Controller.php:443]: would be great to sync up on this - seems like it duplicates a lot of logic - wonder if there's a more streamlined way to achieve the same result?

File: app/Http/Controllers/Controller.php (hunk 1 above)
Comment (anchovie91471) [reply app/Http/Controllers/Controller.php:105]: One of these days, I'm gonna get all my environments right! Thanks for catching!

File: resources/views/home.blade.php [hunk 3]
```diff
@@ -24,7 +24,13 @@ function setColorMode(dark) {
     <main class="container-md my-5">
         <div class="row">
             <div class="col-md-6 offset-md-3">
-                {{ html()->form('GET', '/pdf')->attribute('accept-charset', 'UTF-8')->open() }}
+                @php
```
Comment (joshreisner) [inline resources/views/home.blade.php:27]: mind reverting this change, since it's not needed now?

**Comment by joshreisner** [comment]
nice thanks @anchovie91471 ! i think your checked "Page breaks after groups" option is more intuitive than if you'd gone the easier route and made it unchecked "Suppress breaks after groups" (or similar)

File: resources/views/home.blade.php (hunk 3 above)
Comment (anchovie91471) [reply resources/views/home.blade.php:27]: Yep. Thanks for catching!

**Review by joshreisner — APPROVED** [review]

**Pull request merged**